packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -111,12 +111,26 @@ export class ListItem
    */
   @property() bordered = false;
 
-  /** When `true`, a close button is added to the component. */
+  /**
+   * When `true`, a close button is added to the component.
+   *
+   * @deprecated Use `collapsible` prop instead.
+   */
   @property({ reflect: true }) closable = false;
```

Comment:
I don't think this should be collapsible. It isn't collapsed, it isn't shown at all.

Reply:
I think closed/closable needs to remain as is.